test/run.d

```diff
@@ -156,6 +157,8 @@ void ensureToolsExists(string[string] env, const TestTool[] tools ...)
     resultsDir.mkdirRecurse;
 
     shared uint failCount = 0;
+    auto taskPool = new TaskPool(jobs);
+    scope(exit) taskPool.finish();
     foreach (tool; tools.parallel(1))
```

Comment:
@rainers did you mean to write:
```D
foreach (tool; taskPool.parallel(tools, 1))
```

Because as it's written, the `taskPool` variable is unused.

Another valid way to change the thread pool size is:
```
defaultPoolThreads = 2:
````

See: https://dlang.org/phobos/std_parallelism#.defaultPoolThreads

Then, everywhere `range.parallel(1)` is used, the taskPool will use 2 threads.

Reply:
Oops, forgot to update the foreach range when copying the taskPool from the other parallel loop.